can-list.js

```diff
@@ -737,7 +734,7 @@ assign(List.prototype, {
 				}
 			});
 		} else {
-			this.splice.apply(this, [0, this.length].concat(makeArray(newList || [])));
+			this.splice.apply(this, [0, this.length].concat(canReflect.toArray(newList || [])));
```

Comment:
Same with this line; we shouldn’t pass an empty array to `canReflect.toArray`

Reply:
Fixed!